book/src/cluster-test-framework.md

```diff
@@ -0,0 +1,117 @@
+# Cluster Test Framework
+
+This document covers the *Cluster Test Framework*.  The CTF is a test harness
+designed for tests that can execute against a local in process cluster and a
+deployed cluster on the network.
+
+## Motivation
+
+The goal of the CTF is to provide a framework for writing tests that are stable
+against changes to the cluster implementation.  Regressions can be captured in
```

Comment:
I don't think "against changes to the cluster implementation" describes the importance of this very well. I think you mean something like "independent of where and how the cluster is deployed".  Likewise, when describing regression testing, how a description of a triage process where those devs deploying testnets can handoff tests that are reproducible in both the production deployment and on dev machines?